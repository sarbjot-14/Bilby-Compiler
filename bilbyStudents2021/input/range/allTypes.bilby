main{

	 mut rangeChar := <#a..#f>;
   
   	print rangeChar $n;
   
    mut myRange := <14..101>;
    
    print myRange $n;
    
    mut rangeFloat := <14.4..101.1>;
    
    print rangeFloat $n;
    

}