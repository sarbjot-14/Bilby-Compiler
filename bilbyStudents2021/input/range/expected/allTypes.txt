<a..f>
<14..101>
<14.400000..101.100000>
